src/Conversion/ONNXToTOSA/ONNXToTOSALegalizeUtils.cpp

```diff
@@ -104,5 +114,33 @@ Value buildRescaleToInt32(PatternRewriter &rewriter, Operation *op,
       input_zp, 0, false, true);
 }
 
+mlir::Value buildOnnxToTosaPaddingConstOp(mlir::PatternRewriter &rewriter,
```

Comment:
Thank you for implementing this, the padding op code I wrote was a bit of a mess before